README.md

```diff
@@ -34,30 +35,43 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+It is a markup tool that allows you to structure your webpage in a more meaningful way
```

Comment:
Awesome! Semantic is more of a concept that allows us use standardized tags to mark up our HTML in a conventional way